api/sirsidynix_authentication_provider.py

```diff
@@ -64,18 +61,7 @@ class Keys:
             "description": _("The client ID that should be used to identify this CM."),
             "required": True,
         },
-        {
-            "key": BasicAuthenticationProvider.TEST_IDENTIFIER,
-            "label": _("Test Identifier"),
-            "description": BasicAuthenticationProvider.TEST_IDENTIFIER_DESCRIPTION_FOR_OPTIONAL_PASSWORD,
-            "required": True,
-        },
-        {
-            "key": BasicAuthenticationProvider.TEST_PASSWORD,
-            "label": _("Test Password"),
-            "description": BasicAuthenticationProvider.TEST_PASSWORD_DESCRIPTION_OPTIONAL,
-        },
-    ]
```

Comment:
These settings are part of `BasicAuthenticationProvider.SETTINGS` so no need for them here.